apps/web/src/pages/blobs.tsx

```diff
@@ -1,20 +1,26 @@
+import { useMemo } from "react";
 import type { NextPage } from "next";
 import NextError from "next/error";
 import { useRouter } from "next/router";
 
 import { getPaginationParams } from "~/utils/pagination";
-import { BlobCard } from "~/components/Cards/SurfaceCards/BlobCard";
-import { PaginatedListLayout } from "~/components/Layouts/PaginatedListLayout";
+import { PaginatedTable } from "~/components/PaginatedTable/PaginatedTable";
+import {
+  blobsTableHeaders,
+  getBlobsTableRows,
+} from "~/components/PaginatedTable/helpers";
 import { api } from "~/api-client";
 import { formatNumber } from "~/utils";
 
 const Blobs: NextPage = function () {
   const router = useRouter();
   const { p, ps } = getPaginationParams(router.query);
```

Comment:
Now that we're using tables and we're able to show more elements using less space. I would modified the default page size to 50

Reply:
Done :+1: